libs/community/langchain_community/document_loaders/blob_loaders/__init__.py

```diff
@@ -4,6 +4,9 @@
 from langchain_core.document_loaders import Blob, BlobLoader
 
 if TYPE_CHECKING:
+    from langchain_community.document_loaders.blob_loaders.cloud_blob_loader import (
+        CloudBlobLoader,
```

Comment:
**Static Code Issue**: Move import 'langchain_community.document_loaders.blob_loaders.cloud_blob_loader.CloudBlobLoader' out of type-checking block. Import is used for more than type hinting. <br> **Fix**: Move the import statement for 'CloudBlobLoader' outside the TYPE_CHECKING block to the top-level imports. <br> **Code Suggestion**: 
 ```
 3a4
 > from langchain_community.document_loaders.blob_loaders.cloud_blob_loader import CloudBlobLoader
 8,10d8
 <     from langchain_community.document_loaders.blob_loaders.cloud_blob_loader import (
 <         CloudBlobLoader,
 <     )
 ```